client/src/app/tabs/PropertiesContainer.js

```diff
@@ -106,11 +166,16 @@ class PropertiesContainerWrapped extends PureComponent {
 
     const { propertiesPanel = {} } = layout;
 
+    console.log(this.props.layout.propertiesPanel);
+
+    console.log(this.state);
+
     this.changeLayout({
       propertiesPanel: {
         ...DEFAULT_LAYOUT,
         ...propertiesPanel,
-        open: !propertiesPanel.open
+        open: !propertiesPanel.open,
+        width: this.state.lastSetWidth
```

Comment:
Don't we already save the properties panel width? Or why do we need this new `lastSetWidth` property?

Reply:
Cf. https://github.com/camunda/camunda-modeler/pull/2642/files#diff-c1604b72c51ee1de51b7d3e6679b0ea58453884e7f2e86e089a1705e1f48a0fcR97

Reply:
Previously, when we opened the properties panel by clicking/toggling, it opened to the min width we set as default (250px). With this, we are able to fix that value and always open to the last set width before closing.

Reply:
Thank you, that makes sense to me 👍